build.gradle

```diff
@@ -224,6 +224,7 @@ project(':iceberg-api') {
 
   dependencies {
     implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    compileOnly "com.google.code.findbugs:jsr305"
```

Comment:
This has license issues and is not allowed. Instead, we use `com.github.stephenc.findbugs`, which is already available.

Reply:
dropping